samples/model-builder/create_and_import_dataset_image_sample.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_create_and_import_dataset_image_sample]
+def create_and_import_dataset_image_sample(
+    project: str, location: str, display_name: str, src_uris: list
```

Comment:
```suggestion
    project: str, location: str, display_name: str, src_uris: List[str]
```

Not sure if this is the standard for samples.

Reply:
Updated to `Union[str, List[str]]`